### ISSUE TITLE
Can't resolve 'lenis/dist/lenis.css'

### ISSUE THREAD

**Issue by aimryoui** [darkroomengineering/lenis#371]
**Describe the bug**
Can't resolve 'lenis/dist/lenis.css'.

**To Reproduce**
I'm using NextJs with `--turbo` flag and here's my `package.json`:

```json
{
  // ...
  "type": "module",
  "scripts": {
    "dev": "next dev --turbo",
    // ...
  },
  "dependencies": {
    "lenis": "^1.1.8",
    // ...
  }
  // ...
}
```

in my `src/app/layout.tsx`

```tsx
import "lenis/dist/lenis.css"

// ...
```

error:

```cmd
https://nextjs.org/docs/messages/module-not-found


 ⨯ ./src/app/layout.tsx:1:1
Module not found: Can't resolve 'lenis/dist/lenis.css'
> 1 | import "lenis/dist/lenis.css"
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

is it because of `"type": "module"` in `package.json`?

**Comment by clementroche** [comment]
i'm fixing it asap

**Comment by aimryoui** [comment]
I tested this in lenis' `package.json` and it works

```json
{
  "exports": {
    ".": {
      "types": "./dist/lenis.d.ts",
      "default": "./dist/lenis.mjs"
    },
    "./react": {
      "types": "./dist/lenis-react.d.ts",
      "default": "./dist/lenis-react.mjs"
    },
    "./snap": {
      "types": "./dist/lenis-snap.d.ts",
      "default": "./dist/lenis-snap.mjs"
    },
    "./lenis.css": "./dist/lenis.css"
  },
}
```

```tsx
import "lenis/lenis.css"

// ...
```

**Comment by clementroche** [comment]
fixed in [v1.1.9](https://github.com/darkroomengineering/lenis/releases/tag/v1.1.9)
